utils/s2n_mem.c

```diff
@@ -73,10 +73,11 @@ static int s2n_mem_cleanup_impl(void)
 
 static int s2n_mem_free_mlock_impl(void *ptr, uint32_t size)
 {
-    int munlock_rc = munlock(ptr, size);
+    /* We do a best-effort `munlock` and ignore any errors during unlocking.
+     * More details at https://github.com/aws/s2n-tls/issues/2803.
```

Comment:
I'd remove the comment tagging the issue. @lrstewart any suggestions?

Reply:
I left it as a TODO -- it feels somewhat unsatisfactory to not handle errors from a function that _could_ return an error.

I could remote it though; we anyway have a GitHub issue to track this. Let's wait to hear from @lrstewart.

Reply:
Idk, we may end up just closing that issue without making any changes, since it's unclear that freeing memory that munlock failed on is actually a problem. I'd remove the TODO.

Reply:
Cool -- I'll remove the TODO.

Feel free to close the issue, I opened it as per your suggestion just to start the discussion.

Reply:
Changed as suggested.